tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('prepare-deploy:devapp', ['build:devapp'], () => {
```

Comment:
Maybe just make it `deploy:devapp` and have it also run the `firebase deploy`? Anyone running this will almost certainly just run the deploy right after.